client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -57,6 +57,11 @@ export default React.createClass( {
 
 	monthlyPrice: function() {
 		const { cost, currency } = this.props.cartItem;
+
+		if ( typeof cost === 'undefined' ) {
+			return this.translate( 'Loading price' );
```

Comment:
I think we just want to return `null` here. Otherwise it repeats "Loading price" on two different lines.
![loading price](https://cloudup.com/cKqWKqW34P4+)